google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -365,10 +366,14 @@ public EnhancedBigtableStub(EnhancedBigtableStubSettings settings, ClientContext
         new HeaderTracerStreamingCallable<>(
             watched, settings.getHeaderTracer(), getSpanName("ReadRows").toString());
 
+    // Check for "received rst stream" exceptions and convert them to retryable ApiExceptions
+    ServerStreamingCallable<ReadRowsRequest, RowT> convertException =
+        new ReadRowsConvertExceptionCallable<>(withHeaderTracer);
+
```

Comment:
I think this should be a lot deeper in the call chain, it modifies the behavior of the exception callable thats created in the raw callable.